sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/webkey/CertificateKeyType.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Defines values for KeyType.
```

Comment:
Defines values for Certificate Key Type.
here and below..

Reply:
fixed